pkgs/development/python-modules/sphinxcontrib-ditaa/default.nix

```
@@ -0,0 +1,39 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, unittestCheckHook
+, setuptools
+, sphinx
+, ditaa
+}:
+
+buildPythonPackage rec {
+  pname = "sphinxcontrib-ditaa";
+  version = "1.0.2";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-V/LhOwWbOP3olYC+ypFqxsp0VrLXBsPd6p3UiQ5fW9M=";
+  };
+
+  # setuptools: need to fetch this dep on behalf of build
+  #             triggered by pyproject=true
+  dependencies = [ setuptools sphinx ditaa ];
```

```suggestion 
  build-system = [ setuptools ];

  dependencies = [ sphinx ditaa ];
```
`setuptools` should go to `build-system`.
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonpackage-parameters-buildpythonpackage-parameters